theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -50,7 +50,7 @@
                                     class="glyphicon glyphicon-pencil icon-button icon-blue"></button>
                         {% endif %}
 
-                        {% if page.perms.delete and cm.can_be_public_or_discoverable %}
+                        {% if page.perms.delete and cm.can_be_published %}
                             {% if cm.resource_type|lower == "collectionresource" and cm.are_all_contained_resources_published or cm.resource_type|lower != "collectionresource" %}
```

Comment:
The CollectionResource should override the ```can_be_published``` property so that we do not have to check this specifically for CollectionResource here.